src/main/webapp/app/shared/layouts/main/main.component.ts

```diff
@@ -1,22 +1,28 @@
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRouteSnapshot, NavigationEnd, NavigationError, Router } from '@angular/router';
+import { Component, Inject, OnInit, Renderer2 } from '@angular/core';
+import { ActivatedRouteSnapshot, NavigationEnd, NavigationError, NavigationStart, Router } from '@angular/router';
 import { JhiLanguageHelper } from 'app/core/language/language.helper';
 import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
 import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
 import { SentryErrorHandler } from 'app/core/sentry/sentry.error-handler';
 import { ThemeService } from 'app/core/theme/theme.service';
+import { DOCUMENT } from '@angular/common';
 
 @Component({
     selector: 'jhi-main',
     templateUrl: './main.component.html',
 })
 export class JhiMainComponent implements OnInit {
+    public showSkeleton = true;
```

Comment:
can we document the purpose of this attribute here in the code and mention standalone components shown in mobile devices? Please refer to the corresponding components

Reply:
I added clarifying comments to both the property and `shouldShowSkeleton`.